.ci/pitest.sh

```diff
@@ -188,8 +188,9 @@ pitest-tree-walker)
 pitest-utils)
   mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
   declare -a ignoredItems=(
-  "CommonUtil.java.html:<td class='uncovered'><pre><span  class=''>                catch (final URISyntaxException ex) {</span></pre></td></tr>"
-  "CommonUtil.java.html:<td class='uncovered'><pre><span  class='survived'>                    throw new CheckstyleException(UNABLE_TO_FIND_EXCEPTION_PREFIX + filename, ex);</span></pre></td></tr>"
+  "CommonUtil.java.html:<td class='covered'><pre><span  class='survived'>            if (filename.charAt(0) != &#39;/&#39;) {</span></pre></td></tr>"
```

Comment:
not acceptable to extend this technical debt, please cover it by UT.

Reply:
@romani I agree, But I can not fix this

Does any know how to create a valid URL from classpath resource  which it's toURI will throw a URISyntaxException ?